chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -1458,11 +1501,11 @@ class F8HPPro(UV17Pro):
     LIST_GPS_UNITS = ['km/h', 'MPH', 'kn']
     LIST_POWERON_DISPLAY_TYPE = ["LOGO", "BATT voltage", "Station ID"]
     LIST_BEEP = ["Off", "On"]
-    LIST_MENU_QUIT_TIME = ["10 sec", "20 sec", "30 sec", "60 sec", "Off"]
+    LIST_MENU_QUIT_TIME = ["Off", "10 sec", "20 sec", "30 sec", "60 sec"]
```

Comment:
Presumably this will make it wrong for people with the original version right? Is there any way we can tell the version apart, either in the image or during clone/ident?

Reply:
Ah, okay cool.